python/cuml/tests/test_linear_model.py

```diff
@@ -538,11 +554,14 @@ def test_logistic_regression_unscaled(dtype, penalty, l1_ratio):
     culog = cuLog(**params)
     culog.fit(X_train, y_train)
 
-    score_test = 0.94
-    score_train = 0.94
+    score_train = culog.score(X_train, y_train)
+    score_test = culog.score(X_test, y_test)
+
+    target(1 / score_train, label="inverse train score")
+    target(1 / score_test, label="inverse test score")
 
-    assert culog.score(X_train, y_train) >= score_train
-    assert culog.score(X_test, y_test) >= score_test
+    assert score_train >= 0.94
```

Comment:
Would you mind adding a comment here (and maybe a TODO) saying that this value is arbitrary and should be replaced by a theoretically-calculated limit at some point?

Reply:
Done.